docs/release_notes/v1.15.6.md

```diff
@@ -117,3 +118,21 @@ Daprd would connect to Scheduler to receive Workflow work before Placement had d
 ### Solution
 
 Ensure that the daprd connects to the Scheduler only after Placement has disseminated, preventing errors and ensuring that the daprd is ready to process Workflow work without contention.
+
+## Fix Scheduler Deadlock under high load
+
+### Problem
+
+Under high load, like with high Workflow through put, a Scheduler would deadlock.
```

Comment:
nit: should we define high load? what are the figures that we consider high load today?

Reply:
It is kind of arbitrary. It happens when the Etcd database server is under resource contention that this happens, which is sort of dependent of the size of the box.